glean-core/python/glean/_process_dispatcher.py

```diff
@@ -123,7 +123,7 @@ def dispatch(cls, func, args) -> Union[_SyncWorkWrapper, subprocess.Popen]:
             # Explicitly pass the contents of `sys.path` as `PYTHONPATH` to the
             # subprocess so that there aren't any module search path
             # differences.
-            python_path = ":".join(sys.path)
+            python_path = (";" if os.name == "nt" else ":").join(sys.path)
```

Comment:
Fly by: 

```suggestion
            python_path = os.pathsep.join(sys.path)
```

This should work across OSs without hardcoding the string, at least in Python3

Reply:
Thank you! @badboy was a few seconds faster it seems.

Reply:
![the-flash-running-electric-power-speed-5bgrw4gq1e4modzz](https://github.com/mozilla/glean/assets/2129/43a7d8dc-816c-4179-9253-dc956423996e)